### PR TITLE
First MVP iteration of "Bright Ethereum Address Service"

### PR DESCRIPTION
**Pull request by Tomasvrba**

### CODE REVIEW

**Comment by Tomasvrba** [comment]
BrightEtehreumSPA

File: src/components/qrGenerator.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React from "react";
+const QRCode = require("qrcode.react");
+
+interface BrightEthereumDeepLinkQRProps {
+  ethAddress: string;
+}
+
+const deepLinkPrefix =
+  "brightid://link-verification/http:%2f%2fnode.brightid.org/Ethereum/";
```
Comment (auryn-macmillan) [inline src/components/qrGenerator.tsx]:
```suggestion
  "brightid://link-verification/http://node.brightid.org/ethereum/";
```

Comment (adamstallard) [reply src/components/qrGenerator.tsx]: Follow the [DFE spec](https://docs.google.com/document/d/1c-AqcoMeRMespeK797D2NK33RV7dm0nKCCbygNwLBak/edit#heading=h.siig9uwm002c)  It should look like `brightid://link-verification/http:%2f%2fnode.brightid.org/ethereum/<address>/`

**Comment by auryn-macmillan** [comment]
This looks really nice @Tomasvrba! Great work.

**Comment by auryn-macmillan** [comment]
If you and @adamstallard are both happy with it, let's merge it then throw it up on github pages serving from [ethereum.brightid.org](ethereum.brightid.org).

**Comment by Tomasvrba** [comment]
Cool, I'll just fix the deep link context and then I'll merge it this evening.

**Pull request merged**